tol_lab_share/config/logging.py

```diff
@@ -1,8 +1,8 @@
-from typing import Any, Dict
+from typing import Any
 
 DEFAULT_LOGGING_LEVEL = "INFO"
 
-LOGGING: Dict[str, Any] = {
+LOGGING: dict[str, Any] = {
```

Comment:
I was assuming type declaration is done uppercase but lowercase seems fine. As the Dict is removed from import statement from typing module, could you confirm using lowercase dict here is right?

Reply:
Since Python 3.9, they've removed the redundancy of having `List` and `Dict` in the `typing` package for the use of generics.  In the past, if you used the lowercase `list` and `dict` you couldn't specify the inner types of those.  That was worked around using the `typing` package, but that's no longer necessary due to improvements to the core Python language, so I got rid of them.

https://peps.python.org/pep-0585/